### ISSUE TITLE
Starting out with julia/nternact using juliaacademy.com, I can't run run code in the notebooks. Help please.

### ISSUE THREAD

**Issue by uncle-phil** [nteract/archived-desktop-app#5404]
Hello guys!

I trying to learn the programming language julia right now. I've found a course on juliaacademy.com. I followed their steps to install Julia, nteract and ijulia, but whenever I try to run the the codes in the notebooks, which I downloaded from the lessons, I get the same error: Execute failed. There are no connected kernels. Yesterday it miraculously worked after trying for a while, which happened again as I type this (I tried to reproduce the error message).

What should I do? Admittedly, I have no idea what kernels even are.

I am on macOS Big Sur running Julia 1.5.3. Thanks for every help.

Phil

**Comment by uncle-phil** [comment]
I got this message now:

Launch kernel failed
A kernel named julia-1.0 does not appear to be available. Try installing the julia-1.0 kernelspec or selecting a different kernel.

And this one:

Execute failed
There is no connected kernel for this content.

**Comment by captainsafia** [comment]
@uncle-phil Can you share how you installed the Julia kernel on your machine?

**Comment by uncle-phil** [comment]
Hello @captainsafia !

I followed this [step-by-step-guide](https://www.youtube.com/watch?v=Yx9krfmUGzQ).

In the meantime, I figured out what I had to do: When I open a notebook, nteract doesn't automatically start the Julia kernel. I need to manually select Runtime -> Julia 1.5.3 from the dropdown and then click Runtime -> Restart. After a couple seconds I am connected.

Thanks for caring to reply!
Have a nice day.

Phil
